File: Core/Entities/Order.cs
using System;
using System.Collections.Generic;

namespace Core.Entities
{
    public class Order
    {
        public int Id { get; set; }
        public string UserId { get; set; }
        public int TotalPrice { get; set; }
        public bool Status { get; set; }
        public DateTime Date { get; set; }
        public ICollection<BasketItem> BasketItems { get; set; }
    }
}

File: Client/src/app/models/order.ts
export interface Order {
  id: number;
  userId: string;
  totalPrice: number;
  discountedPrice: number;
  priceToPay: number;
  couponCode: string;
  status: boolean;
  paymentReceipt: string;
  date: Date;
  salespersonShareAmount: number;
  salespersonSharePaid: boolean;
}
